templates/narrativeqa/templates.yaml

```diff
@@ -0,0 +1,109 @@
+dataset: narrativeqa
+templates:
+  3b53c95c-022b-4a51-946a-6c88b962892a: !Template
+    id: 3b53c95c-022b-4a51-946a-6c88b962892a
+    jinja: 'Synopsis: {{ document.summary.text }}
+
+
+      Answer the question.
+
+
+      {{ question.text }}
+
+
+      Full text: {{ document.text }} |||
+
+      {% set i= range(0, answers|length) | random %}
+
+      {{ answers[i].text }}'
+    name: Template_05
+    reference: Text + Summary
+    task_template: false
+  62df1726-be83-4403-9483-732da1174cf7: !Template
+    id: 62df1726-be83-4403-9483-732da1174cf7
+    jinja: '{{ document.text }}
+
+
+      Using the above text, answer the following question.
+
+
+      {{ question.text }} |||
+
+      {% set i= range(0, answers|length) | random %}
+
+      {{ answers[i].text }}'
```

Comment:
more compact way to write that: `{{answers | map(attribute="text") | list | choice }}`

Reply:
same for all templates